src/Control/IPUtils.php

```diff
@@ -7,7 +7,7 @@
  *
  * @author Fabien Potencier <fabien@symfony.com>
  */
-namespace SilverStripe\Control\Util;
+namespace SilverStripe\Control;
```

Comment:
Would you mind adding a old-to-new mapping entry in `.upgrade.yml` for this change?

Reply:
Done.